InfiniteLoopBookProject/src/com/st/lms/dao/DAOBasic.java

```diff
@@ -0,0 +1,22 @@
+package com.st.lms.dao;
+
+import java.io.IOException;
+
+public interface DAOBasic {

```

Comment:
If you are going to have this over arching interface, you may want to make it generic, which you seem to be doing with Entry.

Reply:
But I think you need to do
```suggestion
public interface DAOBasic<T> {
```
And then change the Entry to T